server/views/match/placementRequests/occupancyView/view.njk

```diff
@@ -42,96 +41,160 @@
+                        Available for your criteria
+                    </li>
+                    <li class="calendar__day govuk-tag--red">
+                        Full or overbooked
+                    </li>
+                </ul>
+            </section>
+
+            <section aria-describedby="calendar-key">
+                {{ occupancyCalendar(calendar) }}
+            </section>
+        {% endif %}
     </section>
 
-    <section>
-        <h2 class="govuk-heading-l">Book your placement</h2>
-
-        <form action="{{ paths.v2Match.placementRequests.spaceBookings.viewSpaces({ id: placementRequest.id }) }}" method="post">
-            <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-            <input type="hidden" name="premisesId" value="{{ premisesId }}" />
-            <input type="hidden" name="premisesName" value="{{ premisesName }}" />
-            <input type="hidden" name="apType" value="{{ apType }}" />
-            <input type="hidden" name="startDate" value="{{ startDate }}" />
-            <input type="hidden" name="durationDays" value="{{ durationDays }}" />
-
-            {{ govukDateInput({
-                id: "arrivalDate",
-                namePrefix: "arrivalDate",
-                fieldset: {
-                    legend: {
-                        text: "Arrival date",
-                        classes: "govuk-fieldset__legend--m"
-                    }
-                },
-                hint: {
-                    text: "For example, 17 5 2024"
-                },
-                items: dateFieldValues('arrivalDate', errors),
-                errorMessage: errors.arrivalDate
-            }) }}
-
-            {{ govukDateInput({
-                id: "departureDate",
-                namePrefix: "departureDate",
-                fieldset: {
-                    legend: {
-                        text: "Departure date",
-                        classes: "govuk-fieldset__legend--m"
-                    }
-                },
-                hint: {
-                    text: "For example, 17 5 2024"
-                },
-                items: dateFieldValues('departureDate', errors),
-                errorMessage: errors.departureDate
-            }) }}
-
-            <div class="govuk-button-group">
-                {{ govukButton({
-                    text: "Continue",
-                    preventDoubleClick: true
+    {% if not errors.startDate %}
+        <section>
+            <h2 class="govuk-heading-l">Book your placement</h2>
+
+            <form action="{{ paths.v2Match.placementRequests.search.occupancy({ id: placementRequest.id, premisesId: premises.id }) }}"
+                  method="post">
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+                <input type="hidden" name="premisesId" value="{{ premises.id }}" />
+                <input type="hidden" name="premisesName" value="{{ premises.name }}" />
```

Comment:
As discussed - many of these hidden fields are not needed I think. Really just the premisesId in the path and the `startDate`,`duration` and `criteria`